### PR TITLE
Added testing guide for feed sources

### PR DESCRIPTION
**Pull request by tuj**
https://leantime.itkdev.dk/#/tickets/showTicket/2328

Added testing guide for feed sources

### CODE REVIEW

File: content_structure/test_guide.md [hunk 1]
```diff
@@ -488,3 +489,39 @@ When it is shared it can be displayed in another tenant, but not edited.
...
+
+Feed sources are the link between external data source and slides. An example is the RSS feed.
+When creating a slide with the RSS template a feed source ("datakilde") needs to be selected.
+After the feed source has been selected it is possible to input the url to the rss feed to show data from.
+The feed source contains the code that calls the external source and converts data to a format the RSS template
+understands.
+
+Feed sources are connected to a tenant. New feed sources can be connected to a tenant in through the
+"Datakilde" link in the navigation.
+
```
Comment (Anna-itk) [inline content_structure/test_guide.md:505]: If you want a new feed source on the list you need to contact a developer

File: content_structure/test_guide.md [hunk 2]
```diff
@@ -488,3 +489,39 @@ When it is shared it can be displayed in another tenant, but not edited.
...
+With the calendar api feed source the administrator should select which locations are connected with the given tenant.
+When creating a slide with the given feed source, the resources that belong to the given locations are available
+to deliver data to the slide.
+
+### Steps
+
+* Go to `/admin/feed-sources/list`.
+* Click "Opret ny datakilde" (`/admin/feed-sources/create`).
+* Select a name (e.g. "My calendar feed") and description for the feed source.
```
Comment (Anna-itk) [inline content_structure/test_guide.md:520]: and make a description of the feed source

File: content_structure/test_guide.md [hunk 3]
```diff
@@ -488,3 +489,39 @@ When it is shared it can be displayed in another tenant, but not edited.
...
+* Selected "Kalender feed" for "Type".
```
Comment (Anna-itk) [inline content_structure/test_guide.md:521]: select

**Review by Anna-itk — CHANGES_REQUESTED** [review]
Some minor comments, mainly on language. Otherwise it looks fine.

File: content_structure/test_guide.md (hunk 1 above)
Comment (tuj) [reply content_structure/test_guide.md:505]: I do not think this is relevant for the guide.

**Pull request merged**